certora/spec/CVL2/timelockAuthorizerTed.spec

```diff
@@ -0,0 +1,556 @@
+import "timelockAuthorizerMain.spec";
+
+
+invariant hasPermissionIfIsGrantedOnTarget(bytes32 id, address account, address where)
+    isPermissionGrantedOnTarget(id, account, where) => hasPermission(id, account, where)
+
+
+// STATUS - verified
+// https://prover.certora.com/output/40577/43649cf8b8f148a98a7fb83a7647cc34/?anonymousKey=f4c1e5e509629a8ad342dc1997b0cc430264cd84
```

Comment:
for passing rules, you can remove links because they give us nothing

Reply:
You said that it is up to my preference and I wanted the links to be there, as it was about 3 times useful for me during the development (I wanted to see the time when the rule has been ran, because the code changed and I needed all rules to be checked again -- after it was changed). It is true, that now I don't need those links anymore. I will delete those now.